src/averageAges.js

```diff
@@ -14,12 +18,14 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const man = people.filter(person => century
+    ? Math.ceil(person.died / 100) === century && person.sex === 'm'
+    : person.sex === 'm'
```

Comment:
Не розривай колбеки, краще вибирай яким колбеком фільтрувати масив

```suggestion
  const man = people.filter(
    century
     ? person => Math.ceil(person.died / 100) === century && person.sex === 'm'
     :  person => person.sex === 'm'
```